frontend/src/views/RegisterView.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<div class="page_box_wrapper">
+		<div class="page_box">
+			<h1>REGISTER</h1>
+			<form
+				method="post"
+				action=""
+				class="c_block c_form_group"
+				@submit.prevent="userStore.register(registerForm)"
+			>
+				<InputField label="username" v-model="registerForm.name" />
+				<InputField label="password" v-model="registerForm.password" />
+				<InputField
+					label="confirm password"
+					v-model="registerForm.password_confirm"
+				/>
+				<div class="c_block c_split">
+					<p><a href="/login">back</a></p>
+					<p>
+						<input
+							class="link_button"
+							action="/settings"
+							type="submit"
+							value="register"
+						/>
+					</p>
+				</div>
+				<div v-if="userStore.errors.length">
+					<p v-for="error in userStore.errors">
+					<p style="color:red;">!! {{ error }}</p>
```

Comment:
sooo in html/css inline styles are considered bad :P 
it's generally better to avoid using `style=""` on any element unless otherwise impossible
better to use something like `class="error"` and add to the `<style>` section `.error { color: red; }`

you open 2 paragraph elements inside of each other
so that is repeated for every error, 3 errors become this:
```html
<p>
    <p class="error">error 1</p>
</p>
<p>
    <p class="error">error 2</p>
</p>
```
You can move the class (/style) attribute together with the v-for:
```vue
<p v-for"error in userStore.errors" class="error">!! {{ error }}</p>
```